lib/forklift/box_distributor.rb

```diff
@@ -63,6 +63,7 @@ def define_vm(config, box = {})
       config.vm.define box.fetch('name'), primary: box.fetch('default', false) do |machine|
         machine.vm.box = box.fetch('box_name')
         config.ssh.insert_key = false if SUPPORT_SSH_INSERT_KEY
+        config.ssh.forward_agent = true
```

Comment:
I'd prefer if this was configurable and off by default. In most cases agent forwarding is not wanted because of the security risk. Please do add the option to ```boxes.yaml.example``` though.